src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/settings/AlertAddressForm.tsx

```diff
@@ -74,22 +71,15 @@ export const AlertAddressForm = (props: Props) => {
     defaultValue: commsValue(),
     onChange: (newValue) => {
       const chosenOption = newValue as EmailUpdateCommTypeOfOptions;
-      const body: EmailUpdateCommOptionRequest = {
-        instantBreachAlerts: chosenOption,
-        monthlyMonitorReport: monitorReportAllowed,
-      };
```

Comment:
If you leave this in and make the change to the type that I suggested in the other file, then if people modify the backend, TS will warn us that we'll have to update the frontend as well.

(In fact, we probably want to do the same for the call to update `monthlyMonitorReport`, since we have no checks that we're matching the request body type there either.)

```suggestion
      const body: EmailUpdateCommOptionRequest = {
        instantBreachAlerts: chosenOption,
      };
```